tools/encoder/src/TTFFontEncoder.h

```diff
@@ -2,9 +2,10 @@
 #include FT_FREETYPE_H
 #include "Font.h"
 
-#define START_INDEX     32
-#define END_INDEX       127
-#define GLYPH_PADDING   4
+#define START_INDEX			32
```

Comment:
Lets rename these all to GLYPH_INDEX_START, GLYPH_INDEX_END, GLYPH_INDEX_EXTENDED